eclair-core/src/main/scala/fr/acinq/eclair/transactions/Scripts.scala

```diff
@@ -105,6 +93,9 @@ object Scripts {
     // @formatter:on
   }
 
+  def scriptPubKeyHtlcSend(ourkey: PublicKey, theirkey: PublicKey, abstimeout: Long, reltimeout: Long, rhash: ByteVector32, commit_revoke: ByteVector): Seq[ScriptElt]
```

Comment:
This doesn't seem consistent with how sigs are handled. Also, it looks like `OP_PUSHDATA` can take a PublicKey as argument?